grpc/execution/server_test.go

```diff
@@ -0,0 +1,650 @@
+package execution
+
+import (
+	astriaGrpc "buf.build/gen/go/astria/execution-apis/grpc/go/astria/execution/v1alpha2/executionv1alpha2grpc"
+	astriaPb "buf.build/gen/go/astria/execution-apis/protocolbuffers/go/astria/execution/v1alpha2"
+	primitivev1 "buf.build/gen/go/astria/primitives/protocolbuffers/go/astria/primitive/v1"
+	sequencerblockv1alpha1 "buf.build/gen/go/astria/sequencerblock-apis/protocolbuffers/go/astria/sequencerblock/v1alpha1"
+	"bytes"
+	"context"
+	"crypto/sha256"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/params"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/credentials/insecure"
+	"google.golang.org/protobuf/types/known/timestamppb"
+	"math/big"
+	"reflect"
+	"testing"
+)
+
+func TestExecutionService_GetGenesisInfo(t *testing.T) {
+	n, ethservice, serviceV1Alpha1 := setupExecutionService(t, 10)
+
+	conn, err := grpc.Dial(GrpcEndpointWithoutPrefix(n), grpc.WithTransportCredentials(insecure.NewCredentials()))
+	if err != nil {
+		t.Fatalf("Failed to dial gRPC: %v", err)
+	}
```

Comment:
is it possible to not actually start the gRPC server but just call methods directly on `serviceV1Alpha1`? that seems like it would work?

Reply:
That would work. I just thought that this would be a more solid integration test where we run all the grpc node setup logic too.